### PR TITLE
[Android] support wake word

### PR DESCRIPTION
**Pull request by KayMKM**
### Purpose
*What is the context of this pull request? Why is it being done?*
support wake word: https://github.com/microsoft/botframework-solutions/issues/1444

### Changes
*Are there any changes that need to be called out as significant or particularly difficult to grasp? (Include illustrative screenshots for context if applicable.)*
1. upgrade speech SDK to 1.8.0
2. add KWS extension because wake word doesn't support in 1.8.0: https://github.com/Azure-Samples/cognitive-services-speech-sdk/issues/457
3. enable wake word and settings

known issue: 
only support "computer" as wake word now. because the custom keyword portal can't generate model now:
https://speech.microsoft.com/portal/d25c19a0873c417b976c92248e1eed7e/CustomKeyword/c4402930-a5d9-42ed-aecb-108b2593d473
image

### Tests
*Is this covered by existing tests or new ones? If no, why not?*

### Feature Plan
*Are there any remaining steps or dependencies before this issue can be fully resolved? If so, describe and link to any relevant pull requests or issues.*
support "machine" as wake word

#### General
- [ ] I have commented my code, particularly in hard-to-understand areas	
- [ ] I have added or updated the appropriate tests	
- [ ] I have updated related documentation

#### Bots
- [ ] I have validated that new and updated responses use appropriate [Speak](https://docs.microsoft.com/en-us/azure/bot-service/dotnet/bot-builder-dotnet-text-to-speech?view=azure-bot-service-3.0) and [InputHint](https://docs.microsoft.com/en-us/azure/bot-service/dotnet/bot-builder-dotnet-add-input-hints?view=azure-bot-service-3.0) properties to ensure a high-quality speech-first experience
- [ ] I have replicated language model changes across the English, French, Italian, German, Spanish, and Chinese `.lu` files and validated that deployment is successful

#### Deployment Scripts
- [ ] I have replicated my changes in the **Virtual Assistant Template** and **Sample** projects
- [ ] I have replicated my changes in the **Skill Template** and **Sample** projects

### CODE REVIEW

**Comment by ryanisgrig** [comment]
@KayMKM is the custom keyword portal issue being tracked or do we need to reach out to the correct team?

File: samples/android/clients/VirtualAssistantClient/directlinespeech/src/main/java/com/microsoft/bot/builder/solutions/directlinespeech/SpeechSdk.java [hunk 1]
```diff
@@ -276,7 +281,7 @@ public void listenOnceAsync(){
         try {
             final Future<Void> task = botConnector.startKeywordRecognitionAsync(KeywordRecognitionModel.fromStream(inputStream,keyword,false ));
             setOnTaskCompletedListener(task, result -> {
-                // your code here
+                LogInfo("keywordRecognitaionStarted");
```
Comment (ryanisgrig) [inline samples/android/clients/VirtualAssistantClient/directlinespeech/src/main/java/com/microsoft/bot/builder/solutions/directlinespeech/SpeechSdk.java]: Nit: rename to startKeywordRecognition

File: samples/android/clients/VirtualAssistantClient/directlinespeech/src/main/java/com/microsoft/bot/builder/solutions/directlinespeech/SpeechSdk.java [hunk 2]
```diff
@@ -291,6 +296,7 @@ public void stopKeywordListening(){
         final Future<Void> task = botConnector.stopKeywordRecognitionAsync();
         setOnTaskCompletedListener(task, result -> {
             // your code here
+            LogInfo("stopKeywordListening");
```
Comment (ryanisgrig) [inline samples/android/clients/VirtualAssistantClient/directlinespeech/src/main/java/com/microsoft/bot/builder/solutions/directlinespeech/SpeechSdk.java]: Nit: rename to stopKeywordRecognition

**Review by ryanisgrig — CHANGES_REQUESTED** [review]
Added logging comments

**Comment by KayMKM** [comment]
@ryanlengel we need to reach out the correct team. I think we can report this issue to the contact email in custom keyword portal. Or if we can find any related github repo?

**Comment by ryanisgrig** [comment]
I'll send an email to them, thanks!

**Review by ryanisgrig — APPROVED** [review]

**Review by bobokids — APPROVED** [review]

**Pull request merged**